Edits to the DAL section (#18)
ivoa-std/architecture: Arch.tex

\section{Data Access Standards}
\label{dal}

The data access standards define APIs for querying and accessing data holdings.
These standards are primarily implemented by data providers so that the community 
can use agreed and shared tools to interact with the data holdings.

As shown in Fig.~\ref{fig:daldeps}, the interconnection of data
access standards is quite complicated, even without showing the general VO landscape 
dependencies. This is due to two main
factors: older standards not yet updated to rely on DALI (Sec.~\ref{dal:dali})
and protocols providing \textit{simple access} (parametric query solutions) to
\textit{relational tablesets} (also supported by TAP, Sec.~\ref{dal:tap}).

Supporting the data access standards, we also have the following standards and specific cases:
\begin{itemize}

> 	\item ADQL: a SQL-based language to allow remote querying of relational databases, including support for astronomy operations;

Reviewer: Is the word _**remote**_ relevant here ?
```suggestion
	\item ADQL: a SQL-based language to allow querying of relational databases, including support for astronomy operations;
```

> 	\item ADQL: a SQL-based language to allow remote querying of relational databases, including support for astronomy operations;

Reviewer: Make this consistent with the description in the ADQL section (line700).
```suggestion
	\item ADQL: a SQL-based language for querying relational tablesets within the VO, including support for astronomy operations;
```
